addons/sms/static/src/messaging/component/message/message_tests.js

```diff
@@ -0,0 +1,172 @@
+odoo.define('sms.messaging.component.MessageTests', function (require) {
+'use strict';
+
+const components = {
+    Message: require('mail.messaging.component.Message'),
+};
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    pause,
+    start: utilsStart,
+} = require('mail.messaging.testUtils');
+
+QUnit.module('sms', {}, function () {
+QUnit.module('messaging', {}, function () {
+QUnit.module('component', {}, function () {
+QUnit.module('Message', {
+    beforeEach() {
+        utilsBeforeEach(this);
+        this.createMessageComponent = async message => {
+            const MessageComponent = components.Message;
+            MessageComponent.env = this.env;
+            this.component = new MessageComponent(null, {
+                messageLocalId: message.localId,
+            });
+            delete MessageComponent.env;
+            await this.component.mount(this.widget.el);
+        };
+        this.start = async params => {
+            let { env, widget } = await utilsStart(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
```

Comment:
empty newline before/after function definitions

Reply:
I'm not sure what do you mean, but all other tests have the same format as this one in that regard?

Reply:
This is the only comment left where I don't know what to do. Please check/specify and you can review again.

Reply:
done